### ISSUE TITLE
`internlm/internlm-xcomposer2-vl-7b` only processed the first element in the batch

### ISSUE THREAD

**Issue by YerongLi** [InternLM/InternLM-XComposer#427]
`interleav_wrap` inside `internlm/internlm-xcomposer2-vl-7b` only processed the first `text_list` within the inner batch_size. 
The output mask size and attention size currently also have batch size 1, which indicates that it only took the first input.
It should process all of them.
```
wrap_embeds.shape: torch.Size([1, 1738, 4096])
wrap_atts.shape: torch.Size([1, 1738])
```
![Screenshot from 2024-08-21 01-35-26](https://github.com/user-attachments/assets/b972e168-9fdd-4634-919a-f1fe0cf0d077)

**Comment by yuhangzang** [comment]
Please read the document of the [2.0 fine-tune code](https://github.com/InternLM/InternLM-XComposer/tree/main/InternLM-XComposer-2.0/finetune#finetuning-faqs), which only support batch size == 1. The official fine-tune code of 2d5 has support batch size > 1.

**Comment by YerongLi** [comment]
842 = 29 x 29, how I can get the img_split for the old modeling `internlm/internlm-xcomposer2-vl-7b`
